PracticeGit/PracticeGit.vcxproj

```diff
@@ -19,11 +19,11 @@
     </ProjectConfiguration>
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="Program.c" />
-    <ClCompile Include="Test.c" />
+    <ClCompile Include="Cash.c" />

```

Comment:
Since this is an interactive program with a user (not a library), a header file is not really required.